### DISCUSSION TITLE
Welcome to dotnet-algorand-sdk Discussions!

### DISCUSSION THREAD

**Discussion by RileyGe** [RileyGe/dotnet-algorand-sdk#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RileyGe** [comment]
If anyone has trouble with dotnet-algorand-sdk, please ask questions here. 
Thanks and Regard.
你也可以用中文提问，我的中文比英文要好，哈哈。

**Comment by Kushalrock** [comment]
Hello there,
I am trying to use this package inside unity and a method that I am using is building out a .net console project and adding all the packages there including dependencies then building the project. Copying all the XML and dll files out of debug folder and pasting it into the "Plugins" folder for unity and deleting the NewtonJSoft.dll file in the package cache of unity. But everytime I do that unity just rebuilt that and I get the same Multiple Precompiled assemblies error
